MediaRecorder.html

```diff
@@ -661,7 +661,8 @@
       <pre class="idl">
         [Constructor (DOMString type, BlobEventInit eventInitDict)]
         interface BlobEvent : Event {
-          readonly attribute Blob data;
+          [SameObject] readonly attribute Blob data;
+          [SameObject] readonly attribute DOMHighResTimeStamp timecode;
```

Comment:
Also, should probably add it to BlobEventInit as well, to keep up with the copy-construction pattern?